src/schema/submission_centric_schema.yaml

```diff
@@ -324,13 +327,16 @@ classes:
       - has individual
     slot_usage:
       name:
-        required: true
+        required: false
+        recommended: true
       type:
         description: >-
           The type of Biospecimen.
         required: false
+        recommended: true
       description:
-        required: true
+        required: false
+        recommended: true
```

Comment:
Other description slots do not have recommended flag. Is it specific to this one?

Reply:
Just to make it consistent with other slots in biospecimen. I'll delete it, if you think it is false.